src/analysis/retail/priest/shadow/CONFIG.tsx

```diff
@@ -3,15 +3,15 @@ import GameBranch from 'game/GameBranch';
 import SPECS from 'game/SPECS';
 import Config, { SupportLevel } from 'parser/Config';
 
-// import CHANGELOG from './CHANGELOG';
+import CHANGELOG from './CHANGELOG';
 
 const config: Config = {
   // The people that have contributed to this spec recently. People don't have to sign up to be long-time maintainers to be included in this list. If someone built a large part of the spec or contributed something recently to that spec, they can be added to the contributors list. If someone goes MIA, they may be removed after major changes or during a new expansion.
   contributors: [DoxAshe],
   branch: GameBranch.Retail,
   // The WoW client patch this spec was last updated.
-  patchCompatibility: '10.2.7',
-  supportLevel: SupportLevel.MaintainedFull,
+  patchCompatibility: '11.0.0',
+  supportLevel: SupportLevel.Foundation,
```

Comment:
If you're planning to maintain shadow you should set this to `MaintainedPartial`

Reply:
I do, I have changed it to `MaintainedPartial`